packages/tpetra/core/src/Tpetra_Details_PackTraits.hpp

```diff
@@ -249,12 +275,18 @@ struct PackTraits {
       // entries of the input array have the correct size.
       const T& val = packValueCount (outBuf(0));
```

Comment:
Just to follow up on our phone conversation: that line should have read `const T& val = outBuf(0);`.  Otherwise, the comment doesn't make sense.  Thanks for pointing this out!